src/pages/configurePatch/configurePatchCore/ConfigureTasks/index.tsx

```diff
@@ -139,6 +148,7 @@ const ConfigureTasks: React.VFC<Props> = ({
     selectedBuildVariants.forEach((v) => {
       if (selectedBuildVariantsCopy?.[v] !== undefined) {
         Object.keys(selectedBuildVariantsCopy[v]).forEach((task) => {
+          if (search !== "" && !task.includes(search)) return;
```

Comment:
I wonder if this should be a regex match or a normal match. But we can address that in a future ticket.